src/NuGet.Indexing/Extraction/MetadataConstants.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Indexing
+{
+    public static class MetadataConstants
```

Comment:
Are these constants used outside this `NuGet.Indexing` library? If not, make it `internal`.